### PR TITLE
Failed to enable HTML5 Notifications Error Dialogs

### PR DESCRIPTION
**Pull request by t3chguy**
Would appreciate better wording for both scenarios

for vector-im/riot-web#2569

### CODE REVIEW

File: src/Notifier.js [hunk 1]
```diff
@@ -131,6 +133,14 @@ const Notifier = {
             plaf.requestNotificationPermission().done((result) => {
                 if (result !== 'granted') {
                     // The permission request was dismissed or denied
+                    const description = result === 'denied'
+                        ? 'Your browser is not permitting this app to send you notifications.'
+                        : 'It seems you didn\'t accept notifications when your browser asked';
```
Comment (ara4n) [inline src/Notifier.js]: I suggest:

"Riot does not have permission to send you notifications - please check your browser settings"

**Comment by ara4n** [comment]
i added a comment for better wording - otherwise lgtm :)

File: src/Notifier.js (hunk 1 above)
Comment (t3chguy) [inline src/Notifier.js]: do you suggest that for both the denied and default (they simply dismissed it) scenarios?

Comment (ara4n) [inline src/Notifier.js]: oops, sorry, i missed this (and misread the ternary as a continuation the first time).  How about this?

denied: "Riot does not have permission to send you notifications - please check your browser settings"
default: "Riot was not given permission to send notifications - please try again"

**Comment by ara4n** [comment]
@t3chguy ptal

**Comment by t3chguy** [comment]
@ara4n PTAL

**Comment by ara4n** [comment]
thanks

**Pull request merged**